apps/coscrad-frontend/src/configurable-front-matter/frontMatterData/sample-front-matter.json

```diff
@@ -0,0 +1,7 @@
+{
+    "siteTitle": "Site Title Here",
+    "subTitle": "Subtitle Here",
+    "about": "About page content here",
+    "siteDescription": "Home page blurb here",
+    "copyrightHolder": "Name of owner organization for instance here."
```

Comment:
We may want to consider being robust to missing data, as long as everything that is there is formatted correctly. I.e., we may want to consider some props as optional (in the data, validation, and presentation).